src/lavinmq/vhost.cr

```diff
@@ -413,6 +417,18 @@ module LavinMQ
       @connections.delete client
     end
 
+    def start_session(client : Client)
+      client_id = client.client_id
```

Comment:
We should prefix the "session name" (= queue name). We have `amq.` as a reserved prefix already, so we could use something like `amq.mqtt-session-#{client_id}`.